paddle/fluid/platform/device_tracer.cc

```diff
@@ -92,15 +96,33 @@ std::string MemcpyKind(CUpti_ActivityMemcpyKind kind) {
   return "MEMCPY";
 }
 
+std::string DriverKind(CUpti_CallbackId cbid) {
+  if (static_cast<size_t>(cbid) >= driver_cbid_str.size())
+    return "Driver API " + std::to_string(cbid);
+  // remove the "CUPTI_DRIVER_TRACE_CBID_" prefix
+  return driver_cbid_str[cbid].substr(24);
+}
+
+std::string RuntimeKind(CUpti_CallbackId cbid) {
+  if (static_cast<size_t>(cbid) >= runtime_cbid_str.size())
+    return "Runtime API " + std::to_string(cbid);
+  // remove the "CUPTI_RUNTIME_TRACE_CBID_" prefix
+  return runtime_cbid_str[cbid].substr(25);
+}
+
 void EnableActivity() {
   // Device activity record is created when CUDA initializes, so we
   // want to enable it before cuInit() or any CUDA runtime call.
   CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_MEMCPY));
-  CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_KERNEL));
-  CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_DEVICE));
-  CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_MEMSET));
-  CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_OVERHEAD));
+  CUPTI_CALL(
+      dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_CONCURRENT_KERNEL));
+  // CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_KERNEL));
+  CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_DRIVER));
+  CUPTI_CALL(dynload::cuptiActivityEnable(CUPTI_ACTIVITY_KIND_RUNTIME));
```

Comment:
What information can be collected by using `CUPTI_ACTIVITY_KIND_DRIVER ` and `CUPTI_ACTIVITY_KIND_RUNTIME `?